pkg/collector/resourceutilization/bpf/process_bpf_collector.go

```diff
@@ -32,14 +32,19 @@ import (
 
 type ProcessBPFMetrics = bpf.ProcessBPFMetrics
 
+const (
+	// convert microseconds to miliseconds
+	MicrosecondsToMiliseconds = 1000
```

Comment:
This shouldn't be package global as its not used anywhere else. Given its only used once I would simply divide by 1000 and put a comment in the line above explaining what you are converting and why

Reply:
done